app/presenters/tree_builder_chargeback.rb

```diff
@@ -13,6 +13,7 @@ def x_get_tree_roots
       {
         :id   => type,
         :text => type,
+        :selectable => false,
```

Comment:
Again, this breaks stuff in the other trees. Please redefine it `TreeBuilderChargeBackRates` and call `super.map` to set them non-selectable.